src/geojson/geojson.ml

```diff
@@ -312,14 +320,15 @@ module Make (J : Intf.Json) = struct
               "A Geojson feature requires the type `Feature`. No type was \
                found.")
 
-    let to_json { id; geometry; props } =
+    let to_json { id; geometry; properties } ?bbox =
       J.obj
-        [
-          ("type", J.string "Feature");
-          ("id", Option.(value ~default:J.null id));
-          ("geometry", Option.(value ~default:J.null @@ map Geometry.to_json t));
-          ("properties", Option.(value ~default:J.null props));
-        ]
+        ([
+           ("type", J.string "Feature");
+           ("id", Option.(value ~default:J.null id));
```

Comment:
You need an equivalent `id_to_json` function like how the `geometry` field is done.

Reply:
I did do a mapping
 ("id", Option.(value ~default:J.null @@ map id_of_json id)); 

![k.png](https://user-images.githubusercontent.com/92878224/164306137-bac4fa7e-e5c6-4ad9-9ae9-55349f006300.png) 


That's what I got

Reply:
Thanks sir